SimpleClient.js

```diff
@@ -1,22 +1,15 @@
-const fetch = require('nodeify-fetch')
-const rdf = require('@rdfjs/data-model')
+const Endpoint = require('./Endpoint')
+const SimpleQuery = require('./SimpleQuery')
 const BaseClient = require('./BaseClient')
-const RawQuery = require('./RawQuery')
 
-class Client extends BaseClient {
-  constructor ({ endpointUrl, factory = rdf, headers, password, storeUrl, updateUrl, user }) {
+class SimpleClient extends BaseClient {
+  constructor (options) {
     super({
-      endpointUrl,
-      factory,
-      fetch,
-      headers,
-      password,
-      storeUrl,
-      updateUrl,
-      user,
-      Query: RawQuery
+      endpoint: new Endpoint(options),
+      factory: options.factory,
+      Query: SimpleQuery
```

Comment:
Related to my other comment, here I'd rather have

```js
class SimpleClient extends BaseClient {
  constructor ({ endpoint, factory }) {
    super({
      endpoint,
      factory,
      Query: SimpleQuery
    })
  }
}
```

And similarly in `StreamClient`